Rakefile

```diff
@@ -129,6 +129,10 @@ task :update do
     chdir app.path
     sh "bin/bundle update"
   end
+
+  # update the shell app as well
+  chdir Pathname.new("#{APPS_DIR}/shell")
+  sh "bin/setup"
```

Comment:
I believe what is wanted for the shell app is `yarn upgrade` though you would need to have run bin/setup first (or otherwise installed yarn) before doing that (and use the yarn that is created in bin/setup